### PR TITLE
Image promotion for artifact-promoter v3.3.0-beta.2-1

### PR DESCRIPTION
**Pull request by justaugustus**
Image promotion for artifact-promoter v3.3.0-beta.2-1
This is an automated PR generated from `kpromo`
```
kpromo pr --fork justaugustus --interactive --projectartifact-promoter --tag v3.3.0-beta.2-1
```

/hold
cc: @kubernetes/release-engineering

### CODE REVIEW

**Comment by justaugustus** [comment]
Testing out the fruits of our labors from https://github.com/kubernetes-sigs/promo-tools/pull/458 and https://github.com/kubernetes-sigs/promo-tools/pull/460.

@CecileRobertMichon -- looks like the PR title was successfully fixed (ref: https://github.com/kubernetes-sigs/promo-tools/pull/458#discussion_r751817059)

/assign @puerco @Verolop

File: k8s.gcr.io/images/k8s-staging-artifact-promoter/images.yaml [hunk 1]
```diff
@@ -12,19 +12,19 @@
     "sha256:768443d88a1c2732adbccdf2d633c35f5a658d92ebf500d0637e2b686e6c239a": ["20200825-v2.3.1-262-g1d9ded6"]
     "sha256:799f81d9a855717eab7198310655d1b55919320b120f0f015c3bb43c4e90316d": ["20190821-v2.2.1-2-gc29e4fc"]
     "sha256:7c672405701dc7decdf98730708506cc64f502a965893cdb441e4943439b6d28": ["20200107-v2.3.1-71-gfd2fae8"]
-    "sha256:883e37aa7651b801a2ed55afc774c2b06caa8756a84f43ed5632dc36a92984c0": ["20201027-pkgv0.3.0-7-gf131ef2", "v2.4.1"]
+    "sha256:883e37aa7651b801a2ed55afc774c2b06caa8756a84f43ed5632dc36a92984c0": ["20201027-pkgv0.3.0-7-gf131ef2","v2.4.1"]
```
Comment (justaugustus) [inline k8s.gcr.io/images/k8s-staging-artifact-promoter/images.yaml:15]: Interesting...

Comment (CecileRobertMichon) [reply k8s.gcr.io/images/k8s-staging-artifact-promoter/images.yaml:15]: was this file authored manually before?

Comment (justaugustus) [reply k8s.gcr.io/images/k8s-staging-artifact-promoter/images.yaml:15]: > was this file authored manually before?

Some aspects of it initially, but this is one of the few manifests that has been using `cip-mm` for a while now.
I _think_ what's happening is the grow manifest functionality is stripping spaces in instances where there are multiple tags.

**Comment by justaugustus** [comment]
Superseded by https://github.com/kubernetes/k8s.io/pull/3171.
/close

File: k8s.gcr.io/images/k8s-staging-artifact-promoter/images.yaml (hunk 1 above)
Comment (justaugustus) [reply k8s.gcr.io/images/k8s-staging-artifact-promoter/images.yaml:15]: xref: https://github.com/kubernetes/k8s.io/pull/3212#discussion_r771604521

**Pull request closed without merging**